Patch-Source: https://sources.debian.org/src/nodejs/20.18.0%2Bdfsg-2/debian/patches/deps/v8-no-static-zlib.patch
--
Description: compile v8 without zlib static lib
 it should have a minor performance impact
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2023-12-16
Forwarded: https://github.com/nodejs/node/issues/33848
--- a/tools/v8_gypfiles/features.gypi
+++ b/tools/v8_gypfiles/features.gypi
@@ -295,7 +295,7 @@
 
     # Compile V8 using zlib as dependency.
     # Sets -DV8_USE_ZLIB
-    'v8_use_zlib%': 1,
+    'v8_use_zlib%': 0,
 
     # Enable Sparkplug
     # Sets -DV8_ENABLE_SPARKPLUG.
--- a/tools/v8_gypfiles/v8.gyp
+++ b/tools/v8_gypfiles/v8.gyp
@@ -1083,7 +1083,6 @@
         'run_torque',
         'v8_internal_headers',
         'v8_maybe_icu',
-        'v8_zlib',
         'v8_pch',
         'fp16',
         'abseil.gyp:abseil',
